app/Services/AgendaApplicationFormService.php

```diff
@@ -131,8 +131,10 @@ public function getExportData(AgendaItem $agendaItem): Collection
             "street",
             "houseNumber",
             "city",
+            "zipcode",
             "email",
             "phonenumber",
+            "birthDay",
```

Comment:
Then, this would need to be changed to the `birthdayFormatted` property instead.